paasta_tools/kubernetes_tools.py

```diff
@@ -786,11 +786,18 @@ def get_uwsgi_exporter_sidecar_container(
     ) -> Optional[V1Container]:
 
         if self.should_run_uwsgi_exporter_sidecar(system_paasta_config):
+            stats_uri_env = V1EnvVar(
+                name="STATS_URI",
+                value=self.get_autoscaling_params().get(
+                    "uwsgi_stats_uri", "http://127.0.0.1:8889/"
+                ),
+            )
```

Comment:
did we go for `STATS_URI` over `STATS_PORT` to not clobber anyone's existing `STATS_PORT` usages? 

it seems pretty unlikely that people will have a stats URI that's not localhost and served on something that's not http for uwsgi purposes - thoughts on just hardcoding the protocol/host and exposing something like `UWSGI_STATS_PORT` so that we hide all the other stuff from our users?

Reply:
I figure this is more flexible, in case we need it for some reason, but yeah maybe YAGNI applies here.

I'm 90% sure the environment variables for this container are independent of the environment in the service's container, so conflicts are not a problem.

Reply:
I mostly just worry about having to add documentation to places that say "add a `uwsgi_stats_uri` key with value `http://127.0.0.1:${YOUR_CUSTOM_PORT}/` (make sure you don't forget the trailing `/`! )" and whatnot - there's a lot less our users can mess up if we just expose the port and there's less documentation for us to write as well :p 

that said: I'm also fine with exposing this - worst case we just do a mass `sed` in the future and switch everyone to `uwsgi_stats_port` if we want to punt on simplyding the config